### ISSUE TITLE
Crash on startup on MacOS Mojave

### ISSUE THREAD

**Issue by pollymow** [flacon/flacon#203]
(maybe similar to https://github.com/iina/iina/issues/1165 ?)
```
Process:               Flacon [1107]
Path:                  /Applications/Flacon.app/Contents/MacOS/Flacon
Identifier:            Flacon
Version:               0
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Flacon [1107]
User ID:               504

Date/Time:             2023-05-19 14:47:00.050 +0200
OS Version:            Mac OS X 10.14.6 (18G9323)
Report Version:        12
Anonymous UUID:        06524EEF-0F74-511B-9442-22751CEC14D8


Time Awake Since Boot: 290 seconds

System Integrity Protection: disabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @executable_path/../Frameworks/libuchardet.0.dylib
  Referenced from: /Applications/Flacon.app/Contents/MacOS/Flacon
  Reason: no suitable image found.  Did find:
	/Applications/Flacon.app/Contents/MacOS/../Frameworks/libuchardet.0.dylib: cannot load 'libuchardet.0.dylib' (load command 0x80000034 is unknown)
	/Applications/Flacon.app/Contents/MacOS/../Frameworks/libuchardet.0.dylib: stat() failed with errno=1
	/Applications/Flacon.app/Contents/MacOS/../Frameworks/libuchardet.0.dylib: cannot load 'libuchardet.0.dylib' (load command 0x80000034 is unknown)
	/Applications/Flacon.app/Contents/MacOS/../Frameworks/libuchardet.0.dylib: stat() failed with errno=1

Binary Images:
       0x10093c000 -        0x100c47ffb +Flacon (0) <4735A375-8730-3EDA-AA9A-AF5B47A00BD4> /Applications/Flacon.app/Contents/MacOS/Flacon
       0x1081ba000 -        0x10822470f  dyld (655.1.1) <4842B61A-B868-3146-9CA9-5582C4198B25> /usr/lib/dyld

Model: MacPro5,1, BootROM 144.0.0.0.0, 6 processors, 6-Core Intel Xeon, 3,06 GHz, 32 GB, SMC 1.39f5
Graphics: AMD Radeon R9 280X, AMD Radeon R9 280X, spdisplays_pcie_device, 3 GB
Memory Module: DIMM 1, 8 GB, DDR3 ECC, 1333 MHz, 0x80AD, 0x484D54333147523743465234412D50422020
Memory Module: DIMM 2, 8 GB, DDR3 ECC, 1333 MHz, 0x80AD, 0x484D54333147523743465234412D50422020
Memory Module: DIMM 3, 8 GB, DDR3 ECC, 1333 MHz, 0x80AD, 0x484D54333147523743465234412D50422020
Memory Module: DIMM 4, 8 GB, DDR3 ECC, 1333 MHz, 0x80AD, 0x484D54333147523743465234412D50422020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x111), Broadcom BCM43xx 1.0 (7.77.61.3 AirPortDriverBrcmNIC-1305.12)
Bluetooth: Version 6.0.14d13, 3 services, 18 devices, 1 incoming serial ports
Network Service: Ethernet 2, Ethernet, en1
PCI Card: pci15b7,5009, NVM Express Controller, Slot-2
```

**Comment by SokoloffA** [comment]
Looks like the cause of the problem is described in [this article](https://medium.com/geekculture/how-ios-15-makes-your-app-launch-faster-51cf0aa6c520). **In short, Apple has changed the format of dynamic libraries.**

Unfortunately, it's not only libuchardet (it's just the first problematic library) as well as a large list of libraries.

 * Flacon.app/Contents/Frameworks/libfreetype.6.dylib
 * Flacon.app/Contents/Frameworks/libglib-2.0.0.dylib
 * Flacon.app/Contents/Frameworks/libgthread-2.0.0.dylib
 * Flacon.app/Contents/Frameworks/libintl.8.dylib
 * Flacon.app/Contents/Frameworks/libjpeg.8.dylib
 * Flacon.app/Contents/Frameworks/libpcre2-16.0.dylib
 * Flacon.app/Contents/Frameworks/libpcre2-8.0.dylib
 * Flacon.app/Contents/Frameworks/libpng16.16.dylib
 * Flacon.app/Contents/Frameworks/libtag.1.dylib
 * Flacon.app/Contents/Frameworks/libtiff.5.dylib
 * Flacon.app/Contents/Frameworks/libuchardet.0.dylib
 * Flacon.app/Contents/Frameworks/libzstd.1.dylib

I will be able to rebuild libuchardet and libtag, but rebuilding libgthread,  libglib an so will cause big problems. 
I'll think about what I can do, but I'm afraid Mojave is too old a system.

**Comment by pollymow** [comment]
OK, thank you. Thought so.
But you might want to make sure that the app cannot be launched on system versions that do not support it, by editing the respective entries in the plist.

**Comment by SokoloffA** [comment]
Of course plist should be fixed, I didn't know there were such problems.

**Comment by SokoloffA** [comment]
Included in release 11.1.0
